src/main.js

```diff
@@ -119,6 +119,27 @@ let load = loader({
     },
   },
 
+  declare: {
+    requires: ['profile', 'cfg', 'WorkerType'],
+    setup: async ({profile, cfg, WorkerType}) => {
+      let queue = new taskcluster.Queue({credentials: cfg.taskcluster.credentials});
+
+      if (profile !== 'production' && profile !== 'staging') {
+        return;
+      }
+
+      await queue.declareProvisioner(cfg.app.provisionerId, {
+        stability: process.NODE_ENV === 'production' ? 'stable' : 'experimental',
+        expires: taskcluster.fromNow('36 hours'),
+        description: cfg.app.description,
+      });
+
+      await WorkerType.scan({}, {
```

Comment:
This is unneeded.  We know that this call has happened when the worker type was created, so we shouldn't need to call this again.

Reply:
The intent is to "refresh" this data periodically (here, roughly daily based on dyno cycling).  Similar to how we "refresh" the docs, schemas, and references.